tools/autoupdate_app_sources/autoupdate_app_sources.py

```diff
@@ -44,11 +54,30 @@
 
 
 @cache
-def get_github() -> tuple[Optional[tuple[str, str]], Optional[github.Github], Optional[github.InputGitAuthor]]:
+def get_github() -> tuple[
+    Optional[tuple[str, str]],
+    Optional[github.Github],
+    Optional[github.InputGitAuthor],
+]:
     try:
-        github_login = (REPO_APPS_ROOT / ".github_login").open("r", encoding="utf-8").read().strip()
-        github_token = (REPO_APPS_ROOT / ".github_token").open("r", encoding="utf-8").read().strip()
-        github_email = (REPO_APPS_ROOT / ".github_email").open("r", encoding="utf-8").read().strip()
+        github_login = (
+            (REPO_APPS_ROOT / ".github_login")
+            .open("r", encoding="utf-8")
+            .read()
+            .strip()
+        )
+        github_token = (
+            (REPO_APPS_ROOT / ".github_token")
+            .open("r", encoding="utf-8")
+            .read()
+            .strip()
+        )
+        github_email = (
+            (REPO_APPS_ROOT / ".github_email")
+            .open("r", encoding="utf-8")
+            .read()
+            .strip()
+        )
```

Comment:
Ye no

Reply:
Can you be a bit more verbose in your review please? I'm not sure to understand what you mean here :sweat_smile:

Reply:
Yeah, this makes code a bit unreadable to have this split on this many lines. With a longer line length, black will leave this untouched.

Reply:
How long?